### ISSUE TITLE
how to decide the value of λ

### ISSUE THREAD

**Issue by silentsaber** [caogang/wgan-gp#60]
Since the **d_loss = fake-real+gp**,  how to decide the value of ***LAMBDA*** to get a reasonable gradient penalty.
I mean, if **fake=-10**, **real=10**,  is **gp=10** appropriate？or the **gp** should be smaller(e.g.  **gp=5**)?

**Comment by hangexuexi** [comment]
i have the same question about it, did you solve this problem? at the start of my train, the wass loss is very small, about 0.01, so how should i set the lambda?

**Comment by cjblocker** [comment]
[The original paper](https://arxiv.org/abs/1704.00028) suggests sticking with 10
> **Penalty coefficient** All experiments in this paper use λ = 10, which we found to work well across a variety of architectures and datasets ranging from toy tasks to large ImageNet CNNs.
